File: JAVA/cxf-server-https/src/main/java/com/chuidiang/ejemplos/cxf/Calculadora.java
package com.chuidiang.ejemplos.cxf;

/**
 * iMARE INDRA (C) 2020 Indra Sistemas (unless otherwise noted). All rights reserved.
 *
 * @author fjabellan
 * @date 18/09/2020
 */
import javax.jws.WebService;

@WebService
public interface Calculadora {
    double suma (double a, double b);
    double resta (double a, double b);
    double multiplica (double a, double b);
    double divide (double a, double b);
}

File: PYTHON/curso-python/08-Modulos/calculadora.py
def suma(a,b):
    return a+b

def resta(a,b):
    return a-b

def multiplica(a,b):
    return a*b

def divide(a,b):
    return a/b

def main():
    a = input ('Dime el primer sumando ')
    b = input ('Dime el segundo sumando ')
    print ('La suma es ', suma(int(a),int(b)))

if (__name__ == '__main__'):
    main()
